langchain/src/agents/agent.ts

```diff
@@ -223,6 +223,7 @@ export abstract class Agent {
     data: SerializedAgent & { llm?: BaseLLM; tools?: Tool[] }
   ): Promise<Agent> {
     switch (data._type) {
+      // todo
```

Comment:
Yeah this is there because Im not sure how to fix this hardcoded type
https://github.com/hwchase17/langchainjs/blob/3e868f3eb5bc1d7f8719284f1d34ecce74be2907/langchain/src/agents/agent.ts#L49